lib/pbench/agent/meister/base.py

```diff
@@ -0,0 +1,15 @@
+from pbench.agent.base import BaseCommand
+from pbench.agent.meister.client import ClientMixIn
+from pbench.agent.meister.meister import MeisterMixIn
+from pbench.agent.meister.start import StartMixIn
+from pbench.agent.meister.stop import StopMixIn
+
+
+class MeisterCommand(BaseCommand, ClientMixIn, MeisterMixIn, StartMixIn, StopMixIn):
+    def __init__(self, context):
+        super(MeisterCommand, self).__init__(context)
```

Comment:
Do we need this the python 2 way? We can use shorter version of super with python 3, I think something like `super().__init__(context)`